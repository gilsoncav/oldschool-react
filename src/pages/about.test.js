import About from './about';

describe('About page component', () => {
  it('renders correctly', () => {
    const wrapper = shallow(<About />);
    expect(wrapper).toMatchSnapshot();
  });
});
