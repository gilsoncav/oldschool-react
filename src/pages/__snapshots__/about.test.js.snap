// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`About page component renders correctly 1`] = `
<div
  className="clean-card text-center"
  style={
    Object {
      "margin": "5vh",
    }
  }
>
  <h1
    className="text-primary text-big"
  >
    This project is a boilerplate for developing with React and its most used toolchain.
  </h1>
  <h1
    className="text-primary text-big"
  >
    Author: 
    <a
      href="https://github.com/kevinfaguiar"
    >
      @kevinfaguiar
    </a>
  </h1>
</div>
`;
